### ISSUE TITLE
Please consider renaming the "Restore" settings on Screen 70-TuneFWRetract

### ISSUE THREAD

**Issue by Thinkersbluff** [CR6Community/Marlin#213]
As discussed under Issue#159, the Retract Override function is designed to allow users to override certain slicer settings during an active print, but some Users may find it difficult to recognize to which specific Slicer settings the override parameters correspond. 
Some Users may also think that the override function is offering functionality that the Slicer does not.

Our User community members use a variety of slicers (Cura, PrusaSlicer, SuperSlicer, IdeaMaker, Slic3r, etc.) and the terms used within those slicers are NOT consistent for the parameters the User can override on this screen.

If Users rely solely upon the labels on this screen to anticipate what each parameter will do, then some Users may think that "Restore", "Length" is the opposite of "Retraction", "Length", rather than the "Extra Prime Distance" from Cura that it actually overrides.  If they do so, they are likely to "mess-up" that print job.

Since the term "Priming" seems to be unique to Cura, changing "Restore" to "Prime" may just change who gets confused...
The term "De-Retraction" seems like a made-up word, but it may also be intuitive for Users to translate into the equivalent setting in whatever slicer they use.
To avoid misinterpreting the "Length" parameter under Restore/Prime/De-Retract - which should probably remain zero in a lot of scenarios - I recommend including that Cura concept of labeling it as an "extra" amount of filament to be added at the end of the full retract/de-retract cycle.

This sketch suggests one alternative arangement for this screen: 

![70-ALTTuneFWRetract](https://user-images.githubusercontent.com/36551518/110254289-9648bc00-7f5c-11eb-8528-72914ee6f13b.jpg)

**Comment by Sebazzz** [comment]
Seems like a good suggestion. Will need to wait for someone with Photoshop access. I can't design screens nor work with Photoshop.

**Comment by Thinkersbluff** [comment]
![20BF404D-0FAA-4C03-84A5-22270A8102AE](https://user-images.githubusercontent.com/36551518/110429466-0187b080-8079-11eb-812e-8d2fa8b3ad83.jpeg)
So, I figured out how to Photoshop and how to update 23_Screens.icl.  Adding "Extra" to the Priming Length label required moving that field down a tad, so I had to edit the touch zone and the data field in Project.dgus.  I have no clue how to raise a pull request, but I did upload the bmp, psd and icl files here in the .zip file.

I built this from the Nightly extui downloaded midnight my time, 9 March 2021.

You will notice I went with a much simpler mod than the first idea, trying to keep things easy.  I hope you find this is useable.

[70_TuneFWRetract-GFX.zip](https://github.com/CR6Community/Marlin/files/6106426/70_TuneFWRetract-GFX.zip)
